src/fvm_multicell.hpp

```diff
@@ -194,15 +240,38 @@ class fvm_multicell {
     }
 
 private:
-
     threshold_watcher threshold_watcher_;
 
     /// current time [ms]
-    value_type t_ = value_type{0};
+    value_type t_ = 0;
 
     /// resting potential (initial voltage condition)
     value_type resting_potential_ = -65;
 
+    /// final time in integration round [ms]
+    value_type tfinal_ = 0;
+
+    /// max time step for integration [ms]
+    value_type dt_max_ = 0;
+
+    /// flag: true after a call to `setup_integration()`; reset
+    /// once integration to `tfinal_` is complete.
+    bool integration_running_ = false;
+
+    struct deliverable_event {
+        double time;
+        target_handle handle;
+        double weight;
+    };
+
+    /// events staged for upcoming integration stage
```

Comment:
see comment about: choose to insert either `an` or `the` in front of `upcoming` in this comment.